addons/udes_stock/data/routes.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<odoo>
+  <data>
+    <!-- Replace pull_ids of route_warehouse0_replen by the new procurement rule -->
```

Comment:
Are we actually replacing anything here? It looks more like we're just creating a new route with no rules.

I don't think route_warehouse0_replen exists at the point this is run - so should the comment not be:
```xml
<!-- Add a new route for Replen -->
```